### ISSUE TITLE
Add a tutorial page in addition to the gif

### ISSUE THREAD

**Issue by kindlychung** [dinedal/textql#56]
The gif file is a terrible container of info. It's unsearchable, and impossible to navigate around.

**Comment by dinedal** [comment]
Agreed, it is a great explination for "what does it do" but is not a great resource for continual learning.

Have you checked out the man page? `man textql` if you installed via homebrew.

**Comment by RalphCorderoy** [comment]
Agree with the OP.  I don't wish to waste my overloaded time by watching text dribble through a GIF when I could skim and read written examples far more quickly.  I'm not investigating textql further, not getting as far as bothering to install, and I suspect others have done the same without taking the effort to say why.  :-(

**Comment by goofle** [comment]
+1
I'm in need of some examples of how to use this package inside a Go application.

**Comment by casidiablo** [comment]
`man textql` does not help much. What you need is a clear, concise Getting Started page, with examples.

A gif is OK if you want to show a small feature, but what you have is a full _tutorial_, with several screens, in a container that can't be paused or navigated.

Oh, just realized this is a very old thread. Let me know if you need help with proper documentation.

**Comment by RalphCorderoy** [comment]
man pages often have an EXAMPLES section near the end. Splitting the documentation up is annoying.
